modules/react/collection/stories/Collection.stories.mdx

```diff
@@ -53,13 +55,42 @@ item for selection, maintaining a cursor, or anything else.
 ### Dynamic Items
 
 The `ListBox` also handles a dynamic collection of items. Instead of providing each `ListBox.Item`
-statically, provide a render function instead. The function is called with an `items` value that is
-the same was what's provided to the model. By default, providing items will enable virtualization.
-This example adds a `maxHeight` to ensure overflow. Virtualization uses absolute positioning of each
-item, which could cause problems for popup menus. If your item count is low, pass
-`shouldVirtualize={false}` to disable virtualization.
+statically, provide a render function instead. The function is called with an `item` value that is
+the same as what's provided to the model.
 
-<ExampleCodeBlock code={DynamicItems} />
+#### Registering Items
```

Comment:
@mannycarrera4 [Originally](https://5d854c26ba934e0020f5e98a-bdjnqbgofl.chromatic.com/?path=/docs/features-collections--basic), you had `Registering Items` as an h3 alongside `Dynamic Items`. I moved it to an h4 _under_ `Dynamic Items` since it applies specifically to the dynamic API. I also condensed some of the text and replaced the code snippets with your strings/objects examples. Snippets seemed redundant given the examples.

Reply:
>  By default, providing items will enable virtualization. This example adds a `maxHeight` to ensure overflow. Virtualization uses absolute positioning of each item, which could cause problems for popup menus

Menus default virtualization to `false`.

Also, what issues? It might be useful to describe what issues may be encountered. For example, an HTML container element can render to the width of children, but if the children are not always in the DOM, the min width may not be calculated according to the widest item. A `width` or `minWidth` should be set on virtualized lists.